src/ImuSensor.cc

```diff
@@ -71,6 +71,15 @@ class ignition::sensors::ImuSensorPrivate
   /// \brief Flag for if time has been initialized
   public: bool timeInitialized = false;
 
+  /// \brief Store world heading offset
+  public: ignition::math::Quaterniond headingOffset;
+
+  /// \brief Store world orientation
+  public: WorldFrameEnumType worldFrameType = WorldFrameEnumType::ENU;
+
+  /// \brief Store localization orientation
+  public: WorldFrameEnumType localizationEnum = WorldFrameEnumType::NONE;
```

Comment:
rename `localizationEnum` to `sensorOrientationRelativeTo` and update doc-string to "Frame relative-to which the sensor orientation is reported"